### PR TITLE
Adding scripts to the checkings

### PR DESCRIPTION
**Pull request by svaningelgem**
As mentioned in the other PR, I added the scripts to be checked as well + fixed the issues that came up during these checks.

### CODE REVIEW

File: scripts/tcpperf_server.py [hunk 1]
```diff
@@ -22,14 +22,14 @@ def __init__(self, pause=60, values=1, start_port=8123, processes=2):
         self.port = start_port
         self.processes = processes
 
-    def client(self, n=2000, format_='hello'):
+    def client(self, n_=2000, format_='hello'):
```
Comment (svenkreiss) [inline scripts/tcpperf_server.py:25]: why `n_`?

File: scripts/tcpperf_plot.py [hunk 2]
```diff
@@ -1,9 +1,11 @@
-from collections import namedtuple
 import csv
+from collections import namedtuple
```
Comment (svenkreiss) [inline scripts/tcpperf_plot.py:2]: I would prefer if these stayed in alphabetical order.

**Review by svenkreiss — APPROVED** [review]
Thanks again! This looks good. Just a minor comment. Feel free to squash and merge yourself.

File: scripts/tcpperf_server.py (hunk 1 above)
Comment (svaningelgem) [reply scripts/tcpperf_server.py:25]: n was shadowing n from the outer scope. Would it be better if I change this to "amount_of_connections"?

File: scripts/tcpperf_plot.py (hunk 2 above)
Comment (svaningelgem) [reply scripts/tcpperf_plot.py:2]: pylint or pycodestyle was complaining about this in python 3.9 if I'm not mistaken...

**Comment by svaningelgem** [comment]
> Thanks again! This looks good. Just a minor comment. Feel free to squash and merge yourself.

"Only those with write access to this repository can merge pull requests."

File: scripts/tcpperf_server.py (hunk 1 above)
Comment (tools4origins) [reply scripts/tcpperf_server.py:25]: Would it be better to have a main function to wrap what is currently in the `if __name__ == '__main__'` block? There are quite a lot of variable added to the global scope there indeed

Comment (svaningelgem) [reply scripts/tcpperf_server.py:25]: Done. It's a better solution.

**Comment by svenkreiss** [comment]
Thanks!

**Pull request merged**